src/utils/object-tree-node.ts

```diff
@@ -11,13 +12,24 @@ const objectProxyHandler = {
    */
   get(node: ProxyHandler, key: string): any {
     if (typeof key === 'symbol') {
+      if (key === Symbol.iterator) {
```

Comment:
this feels fishy -- all of this is to make jest's expect work... which I feel like is kinda lying -- cause if you console.log(changeset.get('some array'), you get the ObjectNodeProxy, which.. I think is fine and it makes sense that you'd have to call unwrap on it to get the human-readable value out?